Documentation/ceph-object.md

```diff
@@ -279,3 +279,12 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+
+## Object Multisite
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters.
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+For more information on multisite please read the [ceph multisite overview](ceph-object-multisite.md) for how to run it.
+For more information on the multisite CRDs please read [ceph-object-multisite-crd](ceph-object-multisite-crd.md).
```

Comment:
I'd suggest deleting this line. They can find the CRD docs after they go to the main multisite doc.